prusa/link/web/main.py

```diff
@@ -134,6 +143,58 @@ def api_info(req):
     return JSONResponse(**info)
 
 
+@app.route('/api/v1/status')
+@check_api_digest
+def api_status(req):
+    """Returns telemetric data about printer, job and transfer"""
+    # pylint: disable=unused-argument
+    job = app.daemon.prusa_link.model.job
+    tel = app.daemon.prusa_link.model.latest_telemetry
+    transfer = app.daemon.prusa_link.printer.transfer
+
+    is_printing = job.job_state == JobState.IN_PROGRESS
+    status = {}
+
+    status_printer = {
+        "temp_nozzle": tel.temp_nozzle,
+        "temp_bed": tel.temp_bed,
+        "axis_z": tel.axis_z
+    }
+
+    # X and Y axes data are available only when not printing
+    if job.job_state is JobState.IDLE:
```

Comment:
no, if job.job_state is not PRINTING or BUSY. The axis are relevant when printer is not moving.